packages/minuteman/extra/setup_iptables.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
```

Comment:
For all the bash scripts in DC/OS have a standard set of options we use:
`set -euo pipefail`
`set -x`